### DISCUSSION TITLE
mmatch: multi-dimensional match

### DISCUSSION THREAD

**Discussion by Ma11heuu** [TDAmeritrade/stumpy#1044] [Ideas]
Are there plans for a mmatch function: match for a multi dimensional time-serie?

**Comment by seanlaw** [comment]
@Ma11heuu It's been a while since I've looked at it but I believe that `match` automatically works for multiple dimensions. By default, it takes each independent row of your multidimensional query, `Q`, and computes the distance profile against each corresponding row of your multidimensional time series, `T`, and then the "distance" is computed as the average distance across all dimensions.

**Comment by Ma11heuu** [comment]
Thank you @seanlaw , it seems to work out of the box.
